syntax = "proto3";

package skaimsg.skaimot;

message SkaimotProtoMsg {
    
    uint32 msg_type_id = 1;


}